test/e2e/specs/login.js

```diff
@@ -85,7 +85,8 @@ describe('E2E login', () => {
         await TestApp.logoutRedirect();
       });
 
-      it('can login to social idp using signin widget (with redirect)', async () => {
+      // eslint-disable-next-line jasmine/no-disabled-tests
+      xit('can login to social idp using signin widget (with redirect)', async () => {
```

Comment:
This there a ticket to track this and re-enable the test?

Reply:
https://oktainc.atlassian.net/browse/OKTA-486720